### DISCUSSION TITLE
i don't know how to start ImGui

### DISCUSSION THREAD

**Discussion by Psycho977** [ocornut/imgui#5826] [New Users Build/Link/Run issues ONLY!]
I apologize for the inconvenience, if this is not a question or a problem, please forgive me,I am new to ImGui and I would like to learn how to develop some interfaces, but unfortunately I can hardly do anything, I have a basic experience of C ++ made in school and I tried to do something in the example, I wanted to ask if it is possible to create a project completely from scratch for an interface? I tried to look for tutorials on youtube but I did not find anything .... how could I do? I don't even know how to create the project...

**Comment by mnesarco** [comment]
Have you read the docs?
https://github.com/ocornut/imgui/blob/master/docs/EXAMPLES.md#getting-started

**Reply by Psycho977** [reply]
thanks for the reply, yes i had read them, the only thing i can't understand is how to create a project on visual studio. i only managed to add the examples, it is to modify them, but i have no idea how to create a project from scratch to be able to add the code, and then modify it on visual studio (for ImGui)

**Reply by ocornut** [reply]
That's a visual studio / c++ question not really a dear imgui question. We provide our pre-made examples.

**Reply by Psycho977** [reply]
sorry for the question unfortunately i am a noob and i have no idea how ImGui works

**Reply by nicolasnoble** [reply]
According to what you're saying, it's less about knowing how ImGui works, and more about how to use Visual Studio.
